client/src/components/advancedSearch/DateRangeFilter.tsx

```diff
@@ -183,12 +203,16 @@ export const deserialize = ({ queryKey }, query, key) => {
   const filterValue = {}
 
   if (query[startKey]) {
-    toQueryValue[startKey] = query[startKey]
+    const startVal = query[startKey]
+    toQueryValue[startKey] = startVal
     const lastValues = [LAST_DAY, LAST_WEEK, LAST_MONTH]
-    if (lastValues.indexOf(+query[startKey]) !== -1) {
-      filterValue.relative = query[startKey]
+    if (lastValues.indexOf(+startVal) !== -1) {
+      filterValue.relative = startVal
+    } else if (!isNaN(parseInt(startVal))) {
```

Comment:
Suggest to change to `} else if (utils.isNumeric(startVal)) {`

Reply:
Note however that the addition of this `else if` this breaks several of the Insights, e.g. the Cancelled Engagement Reports insight now displays the search as "Reports filtered on Engagement date: Last 1744581600000 days, State: Cancelled"…